services/movie.py

```diff
@@ -0,0 +1,33 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
```

Comment:
You should specify inner type of QuerySe
```suggestion
               actors_ids: list[int] = None) -> QuerySet[Movie]:
```

Reply:
Fixed